### PR TITLE
Update tests.sh

### PR DESCRIPTION
**Pull request by johnmartin-oracle**
Select dictionary file based on system.

### CODE REVIEW

File: tests.sh [hunk 1]
```diff
@@ -24,6 +24,7 @@ _SCRIPT_PWD="$PWD"
 COMPACT=0
 __tmp_output=""
 only_test=""
+SYSTEM=`uname -s 2>/dev/null` || SYSTEM="unknown"
```
Comment (pirat89) [inline tests.sh]:
```suggestion
SYSTEM=$(uname -s 2>/dev/null) || SYSTEM="unknown"
```

File: tests.sh [hunk 2]
```diff
@@ -270,9 +271,15 @@ create_text() {
   # optional parameter for setting length of text
 
   is_integer "$1" && chars=$1 || chars=100000
+  case "$SYSTEM" in
+    SunOS)
+      DICT_WORDS=/usr/share/lib/dict/words
+      ;;
+    *)
+      DICT_WORDS=/usr/share/dict/words
```
Comment (pirat89) [inline tests.sh:279]:
```
./tests.sh 
./tests.sh: line 283: syntax error near unexpected token `}'
./tests.sh: line 283: `}'
```

```suggestion
      DICT_WORDS=/usr/share/dict/words
      ;;
    esac
```

**Review by pirat89 — CHANGES_REQUESTED** [review]
Thanks for the contribution. Please fix the code as suggested.

**Comment by johnmartin-oracle** [comment]
Sorry for the typo.  I did test the code at home on Jetson Nano running Ubuntu 18.04.
However, instead of uploading the file I typed it directly to the GitHub GUI thinking
I couldn't make a mistake.  :-(

**Comment by pirat89** [comment]
@johnmartin-oracle  no problem :) it happens.

**Review by pirat89 — APPROVED** [review]

**Pull request merged**